src/node_messaging.h

```diff
@@ -110,6 +110,19 @@ class Message : public MemoryRetainer {
   friend class MessagePort;
 };
 
+enum class MessageMode {
+  // Regular messages. Sent and delivered to all recipients.
+  NORMAL,
+
+  // Sticky messages. Like normal, but also delivered as the first message for
+  // new message ports in the sibling group.
+  STICKY,
```

Comment:
If I'm reading the code correctly, only one sticky message per `SiblingGroup` is permitted correct? If so, should indicate that here in the comment so that it's clear.

Reply:
Yes, updated the comment.